docs/data-tunnel/03-tunnel-creation.md

```diff
@@ -3,41 +3,57 @@
 message MsgCreateTunnel {
   option (cosmos.msg.v1.signer) = "creator";
   option (amino.name)           = "tunnel/MsgCreateTunnel";
 
-  repeated SignalInfo signal_infos = 1 [(gogoproto.nullable) = false];
-  google.protobuf.Any route = 2 [(cosmos_proto.accepts_interface) = "Route"];
-  feeds.v1beta1.FeedType feed_type = 3;
-  repeated cosmos.base.v1beta1.Coin deposit = 4
-      [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"];
+  // signal_deviations is the list of signal deviations.
+  repeated SignalDeviation signal_deviations = 1 [(gogoproto.nullable) = false];
+  // interval is the interval for delivering the signal prices in seconds.
+  uint64 interval = 2;
+  // route is the route for delivering the signal prices
+  google.protobuf.Any route = 3 [(cosmos_proto.accepts_interface) = "RouteI"];
+  // initial_deposit is the deposit value that must be paid at tunnel creation.
+  repeated cosmos.base.v1beta1.Coin initial_deposit = 4 [
+    (gogoproto.nullable)     = false,
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins",
+    (amino.dont_omitempty)   = true
+  ];
+  // creator is the address of the creator.
   string creator = 5 [(cosmos_proto.scalar) = "cosmos.AddressString"];
 }
 ```
 
 ## Signal Specification
 
-When creating, the creator provides the list of `SignalInfo` objects to specify the price symbols together with the time interval and price deviation to feed price data.
+When creating, the creator provides the list of `SignalDeviation` objects to specify the price symbols together with the time interval and price deviation to feed price data.
 
-| Field | Description |
-| -- | - |
-| signal_id | The signal id (symbol) to feed price |
-| deviation_bps | Addtional price deviation threshold (in basis point) |
-| interval | The default feeding interval in seconds |
+| Field         | Description                                           |
+| ------------- | ----------------------------------------------------- |
+| signal_id     | The signal id (symbol) to feed price                  |
+| deviation_bps | Additional price deviation threshold (in basis point) |
+| interval      | The default feeding interval in seconds               |
 
-## Tunnel Route and Feed Type
+## Tunnel Route
 
-The creator also specifies the **route** to deliver price data to the destination, currently [TSS](./05-tss-integration.md), [IBC](./06-ibc-intergation.md), and [Axelar](./07-axelar-integration.md) routes are supported. Additionally, the creator specifies the encoding scheme to encode price data as `feed_type`. The current supported values are fixed point decimal encoding and tick encoding.
+The creator also specifies the **route** to deliver price data to the destination, currently [TSS](./06-tss-integration.md), [IBC](./07-ibc-intergation.md), [IBC Hook](./08-ibc-hook-integration.md), and [Router](./09-router-integration.md) routes are supported.
```

Comment:
as we didn't upgrade the testnet yet. move ibc-hook and router to be coming soon first.